common/test/acceptance/tests/lms/test_bookmarks.py

```diff
@@ -84,7 +98,7 @@ def _bookmark_unit(self, course_id, usage_id):
         """ Bookmark a single unit """
         csrftoken = self.session.cookies['csrftoken']
         headers = {'Content-type': 'application/json', "X-CSRFToken": csrftoken}
-        url = LMS_BASE_URL + "/api/bookmarks/v0/bookmarks/?course_id=" + course_id + '&fields=path'
+        url = LMS_BASE_URL + "/api/bookmarks/v1/bookmarks/?course_id=" + course_id + '&fields=path'
```

Comment:
Actually now that the bookmarks button is working, I am assuming these POST requests are going to be replaced by clicks to the button.